libcloud/loadbalancer/drivers/elb.py

```diff
@@ -308,8 +308,7 @@ def ex_create_balancer_listeners(self, name, listeners=None):
             if protocol == 'HTTPS' or protocol == 'SSL':
                 params['Listeners.member.%d.   \
                         SSLCertificateId' % i] = listener[3]
-        else:
-            return False
+        return False
```

Comment:
This one is wrong, I think. You don't want to always return False, but to do so only if `listeners` is empty.